src/uipath/_cli/_evals/mocks/cache_manager.py

```diff
@@ -0,0 +1,91 @@
+"""Cache manager for LLM and input mocker responses."""
+
+import hashlib
+import json
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+
+class CacheManager:
+    """Manages file-based caching for LLM and input mocker responses."""
+
+    _CACHE_DIR = Path.cwd() / ".uipath" / "eval_cache"
+
+    @staticmethod
+    def _compute_cache_key(cache_key_data: Dict[str, Any]) -> str:
+        """Compute a hash from cache key data."""
+        serialized = json.dumps(cache_key_data, sort_keys=True)
+        return hashlib.sha256(serialized.encode()).hexdigest()
+
+    @staticmethod
+    def _get_cache_path(
+        mocker_type: str,
+        eval_set_id: str,
+        eval_item_id: str,
+        cache_key: str,
+        function_name: Optional[str] = None,
```

Comment:
I'd suggest making this not optional. In input mocking, we can use `function_name="get_mock_input"` or something